integration/docker/dockerfile-common.sh

```diff
@@ -55,23 +55,26 @@ function userOperation {
 }
 
 function enableDynamicUser {
-  chmod -R 777 /journal; \
-  chmod -R 777 /mnt; \
-  # Enable user_allow_other option for fuse in non-root mode
-  echo "user_allow_other" >> /etc/fuse.conf; \
+  if [ "$1" = "true" ]; then
```

Comment:
for this function and userOperation function, first check how many inputs?
how many number of parameters do the $@ provides?

Reply:
userOperation is checking the number of parameters I suppose?
Added for the enableDynamicUser